ITL-30 Change style to use Tailwind CSS classes

EmanuelRusuu opened the pull request.

EmanuelRusuu committed:
ITL-31 Change style to use Tailwind CSS classes

- replaced css with tailwind,
- created components for every todo element.

gerebdavid commented on todoapp/src/components/TodoComponents/TodoPriority.vue:
```diff
@@ -0,0 +1,73 @@
+    align-items: center;
+  }
+
+  .Low {
```
Please dont hard code these values

EmanuelRusuu committed:
[ITL-30] Fixed PR issues

- replaced all hard-cored CSS with tailwind,
- improved the code by replacing duplicated with functions, computed and creating layer component classes,
- solved CSS conflicts (!important) with using conditional ternary operator where needed.
